docs/other/payload-validation.mdx

```diff
@@ -0,0 +1,124 @@
+---
+id: payload_validation
+title: Payload validation with AMF
+---
+
+AMF can be used to validate incoming payloads to an API already deployed and running.
+For example, consider a RAML API that declares:
+
+- a `User` schema with `firstname`, `lastname` and `age` properties
+- a `/users` endpoint with an HTTP POST method that requires a `User` object as body parameter
+
+```yaml
+#%RAML 1.0
+title: API with Types
+types:
+  User:
+    type: object
+    properties:
+      firstname: string
+      lastname:  string
+      age:       number
+/users:
+  post:
+    body:
+      application/json:
+        type: User
+```
+
+How can you confirm that a given payload is a valid `User`? You can do that with AMF payload validation.
```

Comment:
To confirm that a given payload is a valid `User`, you can use AMF payload validation. To do so: